components/TopBar/CategoryDescriptions.ts

```diff
@@ -201,13 +201,23 @@ const categoryDescriptions: CategoryDescriptions = {
   'technical writing tools':
     'This list provides the tools to suit your tech writing needs.',
 
+  //DSA
+  'DSA articles':'This section provides various important articles related to DSA problem solving skills, how to tackle them, how to think about it and many more.',
```

Comment:
the description for `DSA articles` has already been added, please remove it or update the existing desc

Reply:
> the description for `DSA articles` has already been added, please remove it or update the existing desc

DONE. @rupali-codes